### PR TITLE
feat: tabbed nft activity details

### LINKED ISSUE #5167: [Task]: Add tabbed approach to NFT activity details
### Task description

Because there is quite a lot of information to potentially display on the NFT activities etc we will move towards a tabbed approach for the activity details. For the NFTs we will potentially have three tabs:

1. Transaction Details
   - Transaction Time
   - Storage Deposit
   - Surplus
   - Tag
   - Mutable Metadata
2. NFT Details
   - NFT Id
   - NFT Address
   - Immutable Issuer
3. Metadata
 Either
  - Raw metadata because it doesn't follow a standard that we support, if JSON we can show as formatted pre text
 OR
  - If irc27 we can show the core key value fields ignoring more complicated structures

### Requirements

- Add tabs to the nft activity details component
- Refactor nft or activity store / interface to support the tab structure

### Creation checklist

- [ ] I have assigned this task to the correct people
- [X] I have added the most appropriate labels
- [X] I have linked the correct milestone and/or project

### PR DESCRIPTION
**Pull request by jeeanribeiro**
## Summary
Adds information tabbed structure to NFT details

## Relevant Issues
closes #5167 

### Platforms
- __Desktop__
  - [ ] MacOS
  - [x] Linux
  - [ ] Windows
- __Mobile__
  - [ ] iOS
  - [ ] Android

## Checklist
- [x] I have followed the contribution guidelines for this project
- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have added or modified tests that prove my changes work as intended
- [x] I have verified that new and existing unit tests pass locally with my changes
- [x] I have verified that my latest changes pass CI workflows for testing and linting
- [ ] I have made corresponding changes to the documentation

### CODE REVIEW

**Review by nicole-obrien — CHANGES_REQUESTED** [review]
Missing info from activity popup:
- Address

Missing some details from the transaction tab:
- Transaction time
- Expiration time
- Claimed status
- Claimed time
- Gifted storage deposit

Missing some info from NFT tab:
- NFT address
- Immutable issuer

File: packages/shared/components/popups/ActivityDetailsPopup.svelte [hunk 1]
```diff
@@ -192,7 +192,7 @@
             <ActivityInformation {activity} />
         </alias-details>
     {:else if activity?.type === ActivityType.Nft}
-        <NftActivityDetails {...details} />
+        <NftActivityDetails activity={{ ...details }} />
```
Comment (MarkNerdi) [inline packages/shared/components/popups/ActivityDetailsPopup.svelte:195]: please use the same approach as with the `Alias`:
```suggestion
        <nft-details class="w-full h-full space-y-6 flex flex-auto flex-col flex-shrink-0">
            <NftActivityDetails {activity} />
            <ActivityInformation {activity} />
        </nft-details>
```

Comment (MarkNerdi) [reply packages/shared/components/popups/ActivityDetailsPopup.svelte:195]: + remove line 101 - 109

File: packages/shared/components/molecules/NftActivityDetails.svelte [hunk 2]
```diff
@@ -163,54 +26,26 @@
...
-                        on:click={handleTransactionIdClick}
-                    >
-                        {truncateString(claimingTransactionId, 12, 12)}
-                    </button>
-                </KeyValueBox>
-            {/if}
-        </details-list>
-    {/if}
-</transaction-details>
+    <ActivityInformation {activity} />
```
Comment (MarkNerdi) [inline packages/shared/components/molecules/NftActivityDetails.svelte:50]: this component should not include the tabs

File: packages/shared/components/popups/send/SendConfirmationPopup.svelte [hunk 3]
```diff
@@ -197,13 +197,16 @@
...
+                    storageDeposit,
+                    subject: recipient,
+                    isInternal,
+                    surplus,
+                    type: ActivityType.Transaction,
+                    direction: ActivityDirection.Outgoing,
+                    inclusionState: InclusionState.Pending,
+                }}
             />
         {/if}
```
Comment (MarkNerdi) [inline packages/shared/components/popups/send/SendConfirmationPopup.svelte:211]: add the tabs here

**Review by MarkNerdi — CHANGES_REQUESTED** [review]
i suggested some changes to correctly adapt my template for the tabs, but if you think your approach has advantages that mine doesnt, we can gladly start a discussion on that

File: packages/shared/components/popups/MintNftConfirmationPopup.svelte [hunk 4]
```diff
@@ -66,7 +66,9 @@
         {localize('popups.mintNftForm.title')}
     </Text>
     <div class="space-y-2 max-h-100 scrollable-y flex-1">
-        <NftActivityDetails nftMetadata={$mintNftDetails} />
+        <NftActivityDetails
+            activity={{ metadata: JSON.stringify($mintNftDetails), direction: ActivityDirection.Outgoing }}
+        />
```
[the remark this answers is not in the document]
Comment (jeeanribeiro) [reply packages/shared/components/popups/MintNftConfirmationPopup.svelte:71]: All tabs, storage deposit is also important here

**Review by nicole-obrien — APPROVED** [review]
LGTM!

**Pull request merged**